finetune/finetune.yaml

```diff
@@ -22,6 +22,7 @@ Training:
   learning_rate: 1.0e-05
   lr_scheduler: linear
   weight_decay: 0.0
+  mixed_precision: bf16
```

Comment:
add a `log_step` option in General part?
This option can be used to control number of training logs for `DefaultTrainer`

Reply:
We should add it. But let's do it in a separate PR.